common/src/main/java/io/netty/util/concurrent/AbstractFuture.java

```diff
@@ -27,23 +28,30 @@
 public abstract class AbstractFuture<V> implements Future<V> {
 
     @Override
-    public V get() throws InterruptedException, ExecutionException {
+    public V get() throws CancellationException, InterruptedException, ExecutionException {
```

Comment:
I wonder if this is a breaking change in terms of API as users may have overriden this method . @trustin @Scottmitch WDYT ?